extension.js

```diff
@@ -33,14 +33,16 @@ const IconGrid = imports.ui.iconGrid;
 const Signals = imports.signals;
 
 // desktop id changed in recent releases
-let ids = ["remmina", "org.remmina.Remmina"];
+let ids = ["org.remmina.Remmina", "remmina", "remmina-file"];
 let remminaApp = null;
 for (let i = 0; !remminaApp && i < ids.length; i++)
 {
-    remminaApp = Shell.AppSystem.get_default().lookup_app(ids[i]);
+    remminaApp = Shell.AppSystem.get_default().lookup_app(ids[i] + ".desktop");
 }
 if (!remminaApp)
+{
     log("Failed to find remmina application");
+}
```

Comment:
Oops, I put these in because there was extra debugging code and then forgot to take them out.